### PR TITLE
feat: API and commands for managing metadata information

### PR DESCRIPTION
**Pull request by gvicentin**

### CODE REVIEW

File: internal/pkg/rpaas/metadata.go [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+		metadata.Annotations = append(metadata.Annotations, item)
+	}
+
+	return metadata, nil
+}
+
+func validateMetadata(items []clientTypes.MetadataItem) error {
+	for _, item := range items {
+		if strings.HasPrefix(item.Name, defaultKeyLabelPrefix) {
+			return &ValidationError{Msg: fmt.Sprintf("metadata key %q is reserved", item.Name)}
```
Comment (wpjunior) [inline internal/pkg/rpaas/metadata.go:62]: Nice @gvicentin

Comment (wpjunior) [reply internal/pkg/rpaas/metadata.go:62]: could you also validate rpaas_instance and rpaas_service labels? Murphy could throw a trap on us

**Review by wpjunior — CHANGES_REQUESTED** [review]
Sounds pretty good =)

File: internal/pkg/rpaas/metadata.go (hunk 1 above)
Comment (gvicentin) [reply internal/pkg/rpaas/metadata.go:62]: yes! it does make sense 😄

Comment (gvicentin) [reply internal/pkg/rpaas/metadata.go:67]: done ✅

**Review by wpjunior — APPROVED** [review]

**Pull request merged**